### PR TITLE
DOCS 3706 adding snmp traps

### PR DESCRIPTION
**Pull request by cswatt**
Adds page on SNMP traps

approved by natalie

### CODE REVIEW

File: content/en/network_monitoring/devices/snmp_traps.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    bind_host: 0.0.0.0
+    users: # limited to only a single v3 user
+      - username: 'user'
+        authKey: 'fakeKey'
+        authProtocol: 'SHA' # choices: MD5, SHA, SHA224, SHA256, SHA384, SHA512
+        privKey: 'fakePrivKey'
+        privProtocol: 'AES' # choices: DES, AES (128 bits), AES192, AES192C, AES256, AES256C
+```
+
+**Note**: Multiple v3 users/passwords are not supported. If this is a requirement in your environment, contact [Datadog support][2].
```
Comment (apigirl) [inline content/en/network_monitoring/devices/snmp_traps.md:41]:
```suggestion
**Note**: Multiple v3 users and passwords are not supported. If this is a requirement in your environment, contact [Datadog support][2].
```

File: content/en/network_monitoring/devices/snmp_traps.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+## Device namespaces
+
+As in [Network Device Monitoring][3], namespaces can be used as tags to differentiate between multiple network devices that may share the same private IP. For example, consider a case of two routers: one in New York, and one in Paris, which share the same private IP. There should be one Agent in the New York datacenter and another in the Paris datacenter. You may wish to tag these with `namespace: nyc` and `namespace: paris`, respectively.
```
Comment (apigirl) [inline content/en/network_monitoring/devices/snmp_traps.md:45]:
```suggestion
As in [Network Device Monitoring][3], namespaces can be used as tags to differentiate between multiple network devices that may share the same private IP. For example, consider a case of two routers: one in New York and one in Paris, which share the same private IP. There should be one Agent in the New York data center and another in the Paris data center. You may wish to tag these with `namespace: nyc` and `namespace: paris`, respectively.
```

File: content/en/network_monitoring/devices/snmp_traps.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+The namespace can then be used to uniquely pivot from an SNMP Trap to the emitter device, or from the emitter device to an SNMP Trap. 
+
+It is critical to have consistency between the multiple Agent configurations. For instance, if you have two Agents configured (for one for trap collection, and the other for metrics) you must ensure that the namespaces exist in both places. Alternatively, ensure that the namespaces exist in neither.
```
Comment (apigirl) [inline content/en/network_monitoring/devices/snmp_traps.md:49]:
```suggestion
It is critical to have consistency between the multiple Agent configurations. For instance, if you have two Agents configured (for example, one for trap collection, and the other for metrics) you must ensure that the namespaces exist in both places. Alternatively, ensure that the namespaces exist in neither. 
```

File: config/_default/menus/menus.en.yaml [hunk 4]
```diff
@@ -2958,31 +2958,36 @@ main:
     parent: nm_parent
     identifier: ndm
     weight: 300
+  - name: SNMP traps
```
Comment (apigirl) [inline config/_default/menus/menus.en.yaml:2961]: should this go at that top? seems like Setup should be first

File: content/en/network_monitoring/devices/snmp_traps.md [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+
+## Device namespaces
+
+As in [Network Device Monitoring][3], namespaces can be used as tags to differentiate between multiple network devices that may share the same private IP. For example, consider a case of two routers: one in New York, and one in Paris, which share the same private IP. There should be one Agent in the New York datacenter and another in the Paris datacenter. You may wish to tag these with `namespace: nyc` and `namespace: paris`, respectively.
+
+The namespace can then be used to uniquely pivot from an SNMP Trap to the emitter device, or from the emitter device to an SNMP Trap. 
+
+It is critical to have consistency between the multiple Agent configurations. For instance, if you have two Agents configured (for one for trap collection, and the other for metrics) you must ensure that the namespaces exist in both places. Alternatively, ensure that the namespaces exist in neither. 
+
+[1]: /monitors/
```
Comment (apigirl) [inline content/en/network_monitoring/devices/snmp_traps.md:51]: links are broken throughout in the preview. You might need a new line here?

**Comment by cswatt** [comment]
i messed this up so don't look at it
new one: https://github.com/DataDog/documentation/pull/14647

**Pull request closed without merging**